tests/Indicator/PhpUnitTestCaseIndicatorTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Indicator;
+
+use PhpCsFixer\Indicator\PhpUnitTestCaseIndicator;
+use PhpCsFixer\Tests\TestCase;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ * @coversNothing
```

Comment:
not sure at this one, cc @keradus

Reply:
why not `$covers \PhpCsFixer\Indicator\PhpUnitTestCaseIndicator` ?

Reply:
sounds good! I'm just not to familiar with these annotations I think